src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,113 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * This class loads and saves a user's tasks
+ */
+public class Storage {
+
+    private static String filePath = "data/list.txt";
```

Comment:
Maybe you should rename it to `FILE_PATH`? Since it's a constant

Reply:
Yes :) I changed it to non-static so that I don't have to rename throughout the file. Thanks!